docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -14,6 +14,93 @@ ms.service: multiple
 
 # Azure CLI 2.0 release notes
 
+## March 13, 2018
+
+Version 2.0.29
+
+### ACR
+
+* Added support for `--image` parameter to `repository delete`
+* Deprecated `--manifest` and `--tag` parameters of the `repository delete` command
+* Added `repository untag` command to remove a tag without deleting data
+
+### ACS
+
+* Added `aks upgrade-connector` command to upgrade an existing connector
+* Changed `kubectl` config files to use a more readable block-style YAML
+
+### Advisor
+
+* [BREAKING CHANGE] Renamed `advisor configuration get` to `advisor configuration list`
+* [BREAKING CHANGE] Renamed `advisor configuration set` to `advisor configuration update`
+* [BREAKING CHANGE] Removed `advisor recommendation generate` 
+* Added `--refresh` parameter to `advisor recommendation list`
+* Added `advisor recommendation show` command
+
+### Appservice
+
+* Deprecated `[webapp|functionapp] assign-identity`
+* webapp/functionapp: author managed identity commands `identity assign/show` [?]
```

Comment:
Not sure what this means.

Reply:
I think 'author' should really be just 'add' which would make it clearer.